start_esp/start_esp.py

```diff
@@ -669,6 +670,10 @@ def make_argparser():
         location in the first few requests. setting this flag to true to skip
         this step.
         ''')
+    parser.add_argument('--disable_trace_sampling', action='store_true',
+        help='''
+        Disable trace sampling.
```

Comment:
Disable cloud trace auto sampling.  By default, 1 request out of every 1000 or 1 request out of every 10 seconds is enabled with cloud trace.  Set this flag to false to disable such auto sampling.  Cloud trace can still be enabled from request HTTP headers with trace context regardless this flag value.

Reply:
Done